Dockerfile

```diff
@@ -33,12 +33,12 @@ CMD ["build"]
 
 ADD ./* /gopath/src/github.com/dennwc/inkview/
 
-ARG CC=arm-none-linux-gnueabi-gcc
+ARG CC=arm-linux-gnueabi-gcc
 ARG GOOS=linux
 ARG GOARCH=arm
-ARG GOARM=5
+ARG GOARM=7
```

Comment:
This is a bit I'm worried about. My device might be older than this. We'll see.

Reply:
Try in pbterm

```bash
cat /proc/cpuinfo
```

For HD3 it displays ARMv7

Reply:
Just tested… Works with `GOARM=5` as well.

Reply:
Yep, you were right, it's ARMv7. Issue solved then, I guess :)